fixtures/Product.yml

```diff
@@ -1,8 +1,8 @@
 entity: Product
 items:
-  productOne{1..100}:
+  productOne{1..10}:
```

Comment:
Unnecessary

Reply:
I launch 5 times yarns test but it never worked with 100 and the current constraint